db/migrate/20191005002659_add_authors_locked_to_boards.rb

```diff
@@ -0,0 +1,18 @@
+class AddAuthorsLockedToBoards < ActiveRecord::Migration[5.2]
+  def up
+    add_column :boards, :authors_locked, :boolean, default: true
+    locked_boards = Board.order(id: :asc).joins(:board_authors).select('boards.id').distinct.pluck('boards.id')
```

Comment:
Do we need anything from Board here? Or can we just pluck all distinct board_ids from BoardAuthor and just use that directly?

Reply:
Hm! that would probably work yes